### PR TITLE
docs for #256

### PR DESCRIPTION
**Pull request by gjo**
I'd already signed in CONTRIBUTORS.txt:124.

### CODE REVIEW

File: docs/basics.rst [hunk 1]
```diff
@@ -607,6 +607,24 @@ __ http://pypi.python.org/pypi/htmllaundry/
                                      preparer=htmllaundry.sanitize,
                                      validator=colander.Length(1))
 
+If you want to create a subclass of `SchemaNode` and define `preparer` as
+its class variable, you'll need to wrap your preparer in `staticmethod`:
```
Comment (stevepiercy) [inline docs/basics.rst]: In reStructuredText, double backticks are for inline code syntax highlighting. Single backticks are for italic or emphasis, unless it is for hyperlinks.

Each of these items needs to have double backticks, e.g., ``SchemaNode``.

While you're at it, would you please also fix the single backticks to double for `preparer` below?

Comment (gjo) [inline docs/basics.rst]: thanks for your review. I'd fix it.

File: docs/basics.rst [hunk 2]
```diff
@@ -607,8 +607,27 @@ __ http://pypi.python.org/pypi/htmllaundry/
                                      preparer=htmllaundry.sanitize,
                                      validator=colander.Length(1))
 
+If you want to create a subclass of :class:`~colander.SchemaNode` and define
+``preparer`` as its class variable, you'll need to wrap your preparer in
```
Comment (stevepiercy) [inline docs/basics.rst]:
```rst
:class:`~colander.SchemaNode`
```
renders as [SchemaNode](http://docs.pylonsproject.org/projects/colander/en/latest/api.html#colander.SchemaNode)

```rst
:class:`colander.SchemaNode`
```

renders as [colander.SchemaNode](http://docs.pylonsproject.org/projects/colander/en/latest/api.html#colander.SchemaNode)

I think the latter is more appropriate in this context.

The other changes are good.

Ping @mmerickel for technical review.

**Comment by mmerickel** [comment]
The issue here is that you're using a function as a method and thus `staticmethod` is required. If you passed a bound method from another class then it would not be necessary. This is just how Python works and I'm not very keen on adding this example to the docs unless it is explained a bit more clearly.

**Comment by mmerickel** [comment]
Closing due to inactivity and these docs are covering a corner case that is not colander-specific.

**Pull request closed without merging**